modules/snooker/src/main/ts/ephox/snooker/api/ResizeBehaviour.ts

```diff
@@ -0,0 +1,144 @@
+import { Arr, Fun } from '@ephox/katamari';
+
+type TableResizer = (delta: number) => void;
+
+export interface ResizeBehaviour {
+  readonly resizeTable: (resizer: TableResizer, delta: number, isLastColumn: boolean) => void;
+  readonly clampTableDelta: (sizes: number[], index: number, delta: number, minCellSize: number, isLastColumn: boolean) => number;
+  readonly calcLeftEdgeDeltas: (input: number[], index: number, nextIndex: number, delta: number, minCellSize: number, relativeSizing: boolean) => number[];
```

Comment:
Oops I forgot to rename `input` here. If you get a chance, mind renaming these to `sizes`?

Reply:
Done

Reply:
Thanks for that :+1: